src/transformers/activations.py

```diff
@@ -121,7 +121,7 @@ class MishActivation(nn.Module):
 
     def __init__(self):
         super().__init__()
-        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9"):
+        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9.1"):
```

Comment:
```suggestion
        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9.0"):
```

Reply:
It seems to have a different result in my env, it returns False
![image](https://user-images.githubusercontent.com/22575232/196795231-f380c8c2-4d0b-48b1-b862-097ca0545a1d.png)

Reply:
Sorry I removed the base version in my case:
```
version.parse(torch.__version__) < version.parse("1.9.0")
```